### PR TITLE
Addition of connectivity, radii and colors.

### PR DESCRIPTION
**Pull request by phohenberger**

### CODE REVIEW

File: znframe/frame.py [hunk 1]
```diff
@@ -40,12 +43,39 @@ class Frame:
...
-    cell: np.ndarray = field(converter=_cell_to_array, eq=cmp_using(np.array_equal))
+    cell: np.ndarray = field(
+        converter=_cell_to_array, eq=cmp_using(np.array_equal), default=np.zeros(3)
+    )
+
+    def __attrs_post_init__(self):
+        if self.connectivity is None:
+            ase_bond_calculator = ASEComputeBonds()
+            self.connectivity = ase_bond_calculator.build_graph(self.to_atoms())
+            self.connectivity = ase_bond_calculator.get_bonds(self.connectivity)
```
Comment (PythonFZ) [inline znframe/frame.py:71]: Could this be a converter instead of being part of the `__post_init__`

**Review by PythonFZ — CHANGES_REQUESTED** [review]
Looks good! There is only one change, where I think the conversion from graph to list should be handled by a converter instead of the inside the post init

**Review by PythonFZ — APPROVED** [review]
Looks good

File: znframe/frame.py (hunk 1 above)
Comment (phohenberger) [reply znframe/frame.py:71]: This is not simply done as you need an `ase.Atoms`-object to calculate the bonds. And it is not possible to convert the `frame` to an `ase.Atoms` while the initialization is still in progess.

**Pull request merged**